aten/src/ATen/native/quantized/library.cpp

```diff
@@ -63,9 +63,6 @@ TORCH_LIBRARY(quantized, m) {
   m.def("conv3d_dilation(__torch__.torch.classes.quantized.Conv3dPackedParamsBase packed_weights) -> int[]");
   m.def("conv3d_groups(__torch__.torch.classes.quantized.Conv3dPackedParamsBase packed_weights) -> int");
   m.def("hardswish(Tensor input, float output_scale, int output_zero_point) -> Tensor");
-  m.def("group_norm(Tensor input, int num_groups, Tensor weight, Tensor bias, float eps, float output_scale, int output_zero_point) -> Tensor");
```

Comment:
For understandind, why are these definitions removed here?

Reply:
yep, the main limitation is that
1. there are kernels and helper functions in c++ needed for this ops which require optional tensors
2. those kernels and helper functions are also used by floating point normalization operators (which are defined in `native_functions.yaml`)
3. the syntax for making a Tensor optional is different between this style of op registration versus `native_functions.yaml`

So, it's either choosing only `native_functions.yaml` or this style for all ops which use the kernels, or writing glue code between the different ways of handling optional tensors.  After discussing with @ezyang we decided that native_functions.yaml is the best solution for this case.